DOCS-7492 scheduler timezone in CloudHub (#1251)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/scheduler-concept.adoc

The Scheduler component enables you to trigger a flow when a time-based condition is met. You can configure it to be triggered at a regular interval or give it a more flexible cron expression. For example, a Scheduler might trigger an event to start a flow every 5 seconds.

> Schedulers follow the timezone used by the machine where the Mule runtime is running. If an application is running in CloudHub, the Scheduler conforms to the UTC timezone, regardless of the geographic deployment region in which the application is running.

Reviewer: Should the second sentence be stated as different from the first, using "however"?
```suggestion
Schedulers use the same timezone as the machine on which Mule is running. However, if an application is running in CloudHub, the Scheduler conforms to the UTC timezone, regardless of the geographic region in which the application is running.
```
